game/world/objects/interactive.cpp

```diff
@@ -3,44 +3,58 @@
 #include <Tempest/Painter>
 #include <Tempest/Log>
 
+#include <phoenix/vobs/mob.hh>
+
 #include "game/serialize.h"
 #include "graphics/mesh/skeleton.h"
 #include "world/objects/npc.h"
 #include "world/world.h"
 #include "utils/dbgpainter.h"
 
-Interactive::Interactive(Vob* parent, World &world, ZenLoad::zCVobData& vob, Flags flags)
+Interactive::Interactive(Vob* parent, World &world, const std::unique_ptr<phoenix::vob>& vob, Flags flags)
```

Comment:
Same as at fireplace.cpp:
reinterpret_cast;
Also, probably it's better to handle cast on caller side and pass phoenix::vobs::mob& here